File: test-data/java/features/Enums.java
package com.example.features;

public enum Enums {
    PENDING("Pending"),
    RUNNING("Running"),
    COMPLETED("Completed"),
    FAILED("Failed");

    private final String displayName;

    Enums(String displayName) {
        this.displayName = displayName;
    }

    public String getDisplayName() {
        return displayName;
    }
}

File: test-data/typescript/features/enums.ts
export enum Status {
    Pending = "PENDING",
    Running = "RUNNING",
    Completed = "COMPLETED",
    Failed = "FAILED"
}

export enum NumericStatus {
    Pending,
    Running,
    Completed,
    Failed
}

export class Task {
    constructor(public name: string, public status: Status = Status.Pending) { }
}
